amd_openvx_extensions/amd_rpp/source/tensor/Brightness.cpp

```diff
@@ -27,8 +27,8 @@ struct BrightnessLocalData {
     Rpp32u deviceType;
     RppPtr_t pSrc;
     RppPtr_t pDst;
-    vx_float32 *pAlpha;
-    vx_float32 *pBeta;
+    Rpp32f *pAlpha;
```

Comment:
why is this changed? It is better to use vx_ types for vx_extensions

Reply:
Most of the members in this structure makes use of Rpp struct definitions and types. Hence in order to maintain uniformity and for better code we have changed all members to be of Rpp type. Please let us know if we can retain the same

Reply:
Done